### DISCUSSION TITLE
raspberry pi3b+ проблема

### DISCUSSION THREAD

**Discussion by paradigmer** [bol-van/zapret#265] [General]
В общем ситуация такова, что если я запускаю скрипт проверки через ssh (curl --connect-to ::speedtest.selectel.ru https://manifest.googlevideo.com/100MB -k -o/dev/null) то пишет что dload = 10m, но если я подключаюсь к точке доступа с телефона и проделываю тоже самое, то там все по нулям. PS Openwrt

**Comment by bol-van** [comment]
через этот девайс идет маршрутизация ip   или там bridge ?
на бридже без модуля ядра br_netfilter   не заработат

**Reply by paradigmer** [reply]
да, в network в разделе config device option type 'bridge'
![image](https://github.com/user-attachments/assets/ee9c9812-6d0b-4c7c-b00b-a3b33f022823)

**Reply by bol-van** [reply]
я имел в виду не работает ли девайс как точка доступа
с отключенным wan

**Reply by paradigmer** [reply]
это я не понимаю где зачекать (

**Reply by bol-van** [reply]
полный файл  /etc/config/network
не картинка

**Reply by paradigmer** [reply]
config interface 'loopback'
option device 'lo'
option proto 'static'
option ipaddr '127.0.0.1'
option netmask '255.0.0.0'

config globals 'globals'
option ula_prefix 'fd90:c183:59f3::/48'

config device
option name 'br-lan'
option type 'bridge'
list ports 'eth0'

config interface 'lan'
option device 'br-lan'
option proto 'static'
option ipaddr '192.168.0.2'
option gateway '192.168.0.1'
option netmask '255.255.255.0'
option ip6assign '60'
list dns '192.168.0.1'
![image](https://github.com/user-attachments/assets/138d1ed4-6553-4048-80e0-e594552ecd3f)

**Reply by ValdikSS** [reply]
Как вы подключаетесь к вашему raspberry с телефона-то?
Вы прописали его как шлюз в настройках роутера? Если да, то см. https://github.com/bol-van/zapret/discussions/262

**Reply by paradigmer** [reply]
Установил на малину openwrt прошивку ну и все вроде бы, в веб интерфейсе настроил вайфай

**Reply by ValdikSS** [reply]
А дальше что? Вы подключаетесь к Wi-Fi-сети Raspberry?
Если вы просто подключили малину в существующую локальную сеть, то она и работает как еще одно устройство локальной сети. Трафик через неё не проходит, разблокировки не применяются.

**Reply by paradigmer** [reply]
Т.е Если она подключена к другому роутеру в этом может быть дело? Мне ее без "посредника" подключать надо? Где-то читал что так тоже можно. Но да я подключаюсь на вайфай малины

**Reply by ValdikSS** [reply]
См. https://github.com/bol-van/zapret/discussions/262 как один из вариантов подключения, у меня такая конфигурация. Преимущество в том, что достаточно поменять один параметр в DHCP, не нужно ни физически ничего переключать, ни перенастраивать.

**Reply by ValdikSS** [reply]
>Но да я подключаюсь на вайфай малины

Тогда должно работать. Вероятно, если всё настроено верно, у вас проблема, описанная по ссылке выше (с которой я столкнулся).

**Reply by bol-van** [reply]
да именно тут все на бридже.  Валдик в своей теме все хорошо описал.

**Reply by paradigmer** [reply]
к сожалению после "opkg install kmod-br-netfilter
sysctl net.bridge.bridge-nf-call-iptables=1" чуда не произошло

**Reply by z0mb1e-kgd** [reply]
@ValdikSS Подскажите, пожалуйста, у меня такая же ситуация? Дано: два роутера Xiaomi 4A Gigabit Edition, на одном из них на входе PPPoE-WAN, установлен OpenWrt 23.05.4, zapret в режиме nftables + nfqws (--dpi-desync=fake,disorder2 --dpi-desync-fooling=md5sig), фильтр autohostlist, WiFi, LAN в бридже провод + WiFi (br-lan). На втором роутере стоковая сяомишная прошивка, он подключен через тупой L2-свитч по проводу в режиме проводного WiFi-репитера к первому роутеру, SSID отличается от SSID на первом роутере. Все проводные клиенты работают с ютубом отлично, а вот у телека, подключенного через WiFi второго роутера, на SmartTube постоянно происходят спонтанные залипы. В доке zapret'а написано, что рекомендуется настроить connbytes при использовании фильтра autohostlist, но я этого не делал (по правде, не знаю, как сделать это грамотно, чтобы не сломать остальной сетап zapret'а, а инструкции в доке по этому поводу я не нашел).
1. Нужно ли мне устанавливать модуль br-netfilter в этой ситуации?
2. Стоит ли всё-таки настроить connbytes? Если да, то как сделать это правильно? Хотя бы в общих чертах.

Заранее спасибо.

**Reply by ValdikSS** [reply]
@z0mb1e-kgd нет, не такая, не надо.
Connbytes на роутерах нужен обязательно.

**Comment by paradigmer** [comment]
upd: сделал все заново, кажется все заработало(как-то криво мб что-то сделал), поэкспереминтировал с blockcheck-результатами. Спасибо bol-van и valdikss

**Reply by paradigmer** [reply]
забавно, но  на яндекс станции макс при подключении к этому роутеру нифига не работает )))))

**Reply by paradigmer** [reply]
ложная тревога, просто она после обновы на "головной вайфай подключилась"
